tests/mpmc_channel.rs

```diff
@@ -1413,4 +1433,209 @@ mod if_std {
             assert_eq!(count, 11);
         }
     }
+
```

Comment:
Those tests all drop the `Future` instead of calling `cancel()` on it? I think those are copies from existing tests, but don't check the new feature?

Reply:
Right, just me being lazy. I'll fix that.